src/amo/pages/Addon/styles.scss

```diff
@@ -165,11 +163,6 @@
         margin-top: 0;
       }
     }
-
-    .AddonDescription-contents,
-    .Addon-developer-comments-contents {
-      max-width: 550px;
```

Comment:
Drive-by fix for https://github.com/mozilla/addons/issues/15431 - made sense to include as more width available made some descriptions slightly shorter in height, and my arbitrary `max-height: 780px` above was picked by trying to still fit a moderately large description, uBlock Origin's, in a very large screen without triggering "Read More".